docs/specs/issues-1654/requirements.md

```
@@ -0,0 +1,116 @@
+- worktree 自体を削除したときの `kill_by_worktree`（`usecase/repository_usecase.rs:243`）。
+
+## 実障害での観測（2026-08-19、PJT-2308 worktree）
+
+再現手順は「同一 worktree で session node を含む workflow を繰り返し実行する」。当日は 02:23 / 02:43 / 08:38 / 08:46 / 10:39 / 10:53 の実行が該当した。
+
+観測された事実:
+
+- 障害時点で、Releash の直接子プロセスのうち cwd が当該 worktree のものがちょうど 32 個残留していた（claude 14、codex/node 17、zsh 1）。
+- 11:46:59 / 11:47:22 に `05_review-fix`（execution `e47d4ec8`）の `create_fix_plan` attempt 2 / 3 が次の出力で失敗した。
+
+  ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)